doofinder-for-woocommerce/includes/api/class-store-api.php

```diff
@@ -235,16 +224,17 @@ private function build_search_engines($api_keys, $primary_language)
             $code = Helpers::format_locale_to_hyphen($code);
             $lang = Helpers::get_language_from_locale($code);
 
+            $home_url = $this->language->get_home_url($lang);
+
             // Prepare search engine body
             $this->log->log('Wizard Step 2 - Prepare Search Enginge body : ');
             $search_engines[] = [
                 'name' => $domain . ($code ? ' (' . strtoupper($code) . ')' : ''),
                 'language' => $code,
                 'currency' => $currency,
-                'site_url' => $this->language->get_home_url($lang),
-                'datatypes' => [
-                    $this->get_datatype($lang)
-                ]
+                'site_url' =>  $home_url,
+                "feed_type" => is_plugin_active('woocommerce/woocommerce.php') ? "product" : "posts", 
+                "callback_url" => $this->build_callback_url($home_url, '/wp-json/doofinder/v1/index-status/?token=' . $this->api_key),
```

Comment:
Just curiosity, but what would happen if the customer did not have the "pretty" permalinks activated (just the simple ones with params)?

Reply:
Well seen, so far it has been working like this and we have not had any cases. I am going to write it down to study it.